src/translations/screens/Profile.ts

```diff
@@ -46,6 +46,11 @@ const ProfileTexts = {
       infoItems: {
         customerNumber: _('Kundenummer', 'Customer number', 'Kundenummer'),
         phoneNumber: _('Telefonnummer', 'Phone number', 'Telefonnummer'),
+        claimsError: _(
+          'Det oppstod et problem med lasting av kontoen din. Trykk for å laste inn på nytt, eller kontakt kundeservice hvis problemet vedvarer.',
```

Comment:
```suggestion
          'Det oppstod et problem med lasting av kontoen din.',
```
Together with @Aliaaen we suggest the above as we think the suggested solution might break with WCAG 1.3.3. Also the second sentence does not add any valuable information to the user - given that customer service can't really do anything ▶️  according to @Sebstorvik .

Reply:
https://github.com/AtB-AS/mittatb-app/pull/4483